amazon-qldb-driver-core/src/retry.rs

```diff
@@ -124,8 +124,8 @@ impl TransactionRetryPolicy for ExponentialBackoffJitterTransactionRetryPolicy {
 }
 
 fn exponential_backoff_with_jitter(base: u32, cap: u32, attempt_number: u32) -> u32 {
-    let max = min(cap, base.pow(attempt_number));
-    thread_rng().gen_range(0..max)
+    let max = min(cap, base * 2_u32.pow(attempt_number));
+    thread_rng().gen_range(1..max)
```

Comment:
I see the other implementations are zero inclusive. Any reason for the change?

Reply:
yeah, last second decision on forcing at least some sort of delay. I've reverted it back to 0 now.